PrismSampleApp/PrismSampleApp/ViewModels/ViewContactListPageViewModel.cs

```diff
@@ -1,22 +1,43 @@
-﻿using Flurl.Http;
-using Prism.Commands;
-using Prism.Common;
-using Prism.Mvvm;
-using Prism.Navigation;
+﻿using Prism.Navigation;
 using PrismSampleApp.Model;
-using PrismSampleApp.Services.Interfaces;
-using System;
 using System.Collections.Generic;
 using System.Collections.ObjectModel;
-using System.ComponentModel;
-using System.Linq;
-using System.Runtime.CompilerServices;
+using System.Windows.Input;
+using Xamarin.Forms;
+using Xamarin.Essentials;
+using PrismSampleApp.Services.Interfaces;
 
 namespace PrismSampleApp.ViewModels
 {
     public class ViewContactListPageViewModel : ViewModelBase , INavigatedAware
     {
         private List<Result> _content;
+        private ObservableCollection<Result> _seperateDetails;
+        private readonly IRandomUserService _randomUserService;
+        public ICommand CallCommand { get; set; }
+        public ICommand SmsCommand { get; set; }
```

Comment:
Use DelegateCommand instead of ICommand throughout the application

Reply:
Uses DelegateCommand throughout the application